graphbolt/src/cuda/neighbor_sampler.cu

```diff
@@ -571,29 +594,59 @@ c10::intrusive_ptr<sampling::FusedSampledSubgraph> SampleNeighbors(
         num_etypes * 2, c10::TensorOptions()
                             .dtype(output_indptr.scalar_type())
                             .pinned_memory(true));
+    auto edge_offsets_dev =
```

Comment:
ditto, I am not sure the diff between edge_offsets and edge_offsets_dev

Reply:
Dev is short for device, one is the pinned tensor, the other is moved to the GPU. Since we need to transfer offsets from the CPU to the GPU, we need to go through pinned to device pattern.